tests/resources/Docker-Util.robot

```diff
@@ -51,6 +53,44 @@ Get Container IP
     Should Be Equal As Integers  ${rc}  0
     [Return]  ${ip}
 
+# Use host docker daemon to start a dind container alongside this test container
+Start Docker Daemon On TCP
+    [Arguments]  ${docker-params}  ${dind-version}=docker:1.13-dind
+    ${rc}  ${id}=  Run And Return Rc And Output  docker run -it --privileged ${docker-params} -d ${dind-version}
+    Should Be Equal As Integers  ${rc}  0
+    [Return]  ${id}
```

Comment:
The id is only use able here like this if you have already done a pull, instead of leaving the pull for other code you should probably have the pull here.